locales/en.json

```diff
@@ -180,5 +180,8 @@
 	"identifies ": "identifies ",
 	" as \"": " as \"",
 	"unblocked ": "unblocked ",
-	"identified ": "identified "
+	"identified ": "identified ",
+	"Nobody will be able to see your posts until you have a follower. The easiest way to get a follower is to use a pub invite as the pub will follow you back. If you have already redeemed a pub invite and it has not followed you back, try another pub.": "Nobody will be able to see your posts until you have a follower. The easiest way to get a follower is to use a pub invite as the pub will follow you back. If you have already redeemed a pub invite and it has not followed you back, try another pub.",
```

Comment:
Looks like there is an unused locale string here.

"Nobody will be able to see your posts until you have a follower. The easiest way to get a follower is to use a pub invite as the pub will follow you back. If you have already redeemed a pub invite and it has not followed you back, try another pub."

Reply:
oh, you fixed this just before I commented 👍